ApiReview/API.Polly.Core/NoDocs/Polly.CircuitBreaker/AdvancedCircuitBreakerStrategyOptions.TResult.cs

```diff
@@ -0,0 +1,18 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.CompilerServices;
+
+namespace Polly.CircuitBreaker;
+
+public class AdvancedCircuitBreakerStrategyOptions<TResult> : CircuitBreakerStrategyOptions<TResult>
+{
+    [Range(0.0, 1.0)]
+    public double FailureThreshold { get; set; }
+    [Range(2, int.MaxValue)]
+    public int MinimumThroughput { get; set; }
+    [TimeSpan("00:00:00.500")]
```

Comment:
Why is the `TimeSpan` value here different from within `AdvancedCircuitBreakerStrategyOptions`?